### YamlMime:JavaMember
uid: "com.azure.resourcemanager.dataprotection.models.InnerError.withEmbeddedInnerError*"
fullName: "com.azure.resourcemanager.dataprotection.models.InnerError.withEmbeddedInnerError"
name: "withEmbeddedInnerError"
nameWithType: "InnerError.withEmbeddedInnerError"
members:
- uid: "com.azure.resourcemanager.dataprotection.models.InnerError.withEmbeddedInnerError(com.azure.resourcemanager.dataprotection.models.InnerError)"
  fullName: "com.azure.resourcemanager.dataprotection.models.InnerError.withEmbeddedInnerError(InnerError embeddedInnerError)"
  name: "withEmbeddedInnerError(InnerError embeddedInnerError)"
  nameWithType: "InnerError.withEmbeddedInnerError(InnerError embeddedInnerError)"
  summary: "Set the embeddedInnerError property: InnerError Child Inner Error, to allow Nesting."
  parameters:
  - description: "the embeddedInnerError value to set."
    name: "embeddedInnerError"
    type: "<xref href=\"com.azure.resourcemanager.dataprotection.models.InnerError?alt=com.azure.resourcemanager.dataprotection.models.InnerError&text=InnerError\" data-throw-if-not-resolved=\"False\" />"
  syntax: "public InnerError withEmbeddedInnerError(InnerError embeddedInnerError)"
  returns:
    description: "the InnerError object itself."
    type: "<xref href=\"com.azure.resourcemanager.dataprotection.models.InnerError?alt=com.azure.resourcemanager.dataprotection.models.InnerError&text=InnerError\" data-throw-if-not-resolved=\"False\" />"
type: "method"
metadata: {}
package: "com.azure.resourcemanager.dataprotection.models"
artifact: com.azure.resourcemanager:azure-resourcemanager-dataprotection:1.0.0-beta.1
